code/modules/atmospherics/atmospherics.dm

```diff
@@ -108,29 +142,70 @@ Pipelines + Other Objects -> Pipe network
 	// Check to see if should be added to network. Add self if so and adjust variables appropriately.
 	// Note don't forget to have neighbors look as well!
 
-	return null
+	// Default behavior is: one network for all nodes in a given dir
+	var/all_nodes = nodes_in_dir(get_dir(src, reference))
```

Comment:
You don't seem to use this, could just throw it into the loop rather than allocating.